parsers/nmparser/summary.go

```diff
@@ -36,38 +35,38 @@ func (results ModelOutput) Summary() bool {
 			rse = math.Abs(seResult / numResult * 100)
 		}
 
-		if rse > 30 {
-			//theta, _ := fmt.Printf("%.3E", results.FinalParameterEstimates.Theta[i])
-			thetaTable.AddRow(
-				aurora.Red("TH "+strconv.Itoa(i+1)),
-				aurora.Red(results.ParameterNames.Theta[i]),
-				aurora.Red(strconv.FormatFloat(numResult, 'f', -1, 64)),
-				aurora.Red(
-					fmt.Sprintf("%s (%s%%)",
-						strconv.FormatFloat(seResult, 'f', -1, 64),
-						strconv.FormatFloat(rse, 'f', 1, 64)),
-				),
-			)
-		} else {
-			thetaTable.AddRow(
-				"TH "+strconv.Itoa(i+1),
-				results.ParameterNames.Theta[i],
-				strconv.FormatFloat(numResult, 'f', -1, 64),
-				fmt.Sprintf("%s (%s%%)",
-					strconv.FormatFloat(seResult, 'f', -1, 64),
-					strconv.FormatFloat(rse, 'f', 1, 64),
-				),
-			)
-
+		if rse > 30.0 {
+			anyRseOver = true
 		}
 
+		thetaTable.Append([]string{
+			string("TH " + strconv.Itoa(i+1)),
+			results.ParameterNames.Theta[i],
+			strconv.FormatFloat(numResult, 'f', -1, 64),
+			fmt.Sprintf("%s (%s%%)", strconv.FormatFloat(seResult, 'f', -1, 64), strconv.FormatFloat(rse, 'f', 1, 64))})
+	}
+
+	if anyRseOver {
```

Comment:
we don't really want the entire column to be red, we'd just want the particular cells to be red.

That said, if some things aren't working (per the commit note), this is low priority and as long as we have tables period thats good enough for now!

Reply:
tablewriter only supports coloring the entire column. 

In the first implementation, the entire column is colored if any of the row values exceed the criterion; in the second iteration(below), only the individual cell is colored. This could be extended to color the entire row.